rmf_charger_msgs/msg/ChargerState.msg

```diff
@@ -0,0 +1,23 @@
+# time when this state message was created
+builtin_interfaces/Time charger_time
+
+uint32 CHARGER_IDLE = 1      # Charger is not occupied
+uint32 CHARGER_ASSIGNED = 2  # Charger has been assigned a robot
+uint32 CHARGER_CHARGING = 3  # Charger is charging
+uint32 CHARGER_ERROR = 200   # Error state, see error_message for info
+
+uint32 state         # One of the previously enumerated states
+
+# the charger name should be unique in the RMF system and
+# should match a charger name appearing in the traffic map
+string charger_name
+
+# the error_message field should be blank unless state is CHARGER_ERROR
+string error_message
+
+# the robot that is currently assigned to this charger (if any)
+string robot_fleet
+string robot_name
+
+# This contains the duration till the robot becomes fully charged.
+builtin_interfaces/Duration time_to_fully_charged
```

Comment:
Is it OK the value keeps changing in the charging process?

Reply:
Yes! This is a state message. It is meant to be used by the charger to provide telemetry updates. Hence all the values in the message can be dynamic.